posts/2015-06-12-best-of-web-2015.md

```diff
@@ -0,0 +1,58 @@
+
+## [Web Audio API](http://ouhouhsami.github.io/2015-06-05-bestofweb-paris/) par Samuel Goldszmidt et Norbert Schnell de l'IRCAM
+Samuel Goldszmidt et Norbert Schnell sont deux chercheurs et développeurs de l'[IRCAM](http://www.ircam.fr/). Ils nous ont fait découvrir les travaux actuellement en cours à l'IRCAM par rapport à l'utilisation des technologies Web et plus précisement la Web Audio API. Leur présentation était intéractive, ils ont commencé par nous montrer quelque travaux réalisés avec la Web Audio API(Amplificateur, filtre, édition de données temporelles, traitement audio). Ils nous ont ensuite présenté le Collective-Soundworks, un framework pour la création d'application audio et multimédia de manière collaborative auquel ils participent et nous on fait une démonstration collaborative en nous demandant de nous connecter à une url pour que l'ensemble de nos terminaux jouent ensemble certaines sonorités.
+Une présentation assez captivante et qui donne beaucoup d'idées d'applications.
+
+## Le [Material design avec Polymer](https://docs.google.com/presentation/d/1IzsxsE6HybPAdbrI8iIlF3Qaj9BQGEHmu3zX7xT9o8M/edit#slide=id.g3a6159c6f_024) par Martin Gorner
+Martin Gorner a commencé sa présentation en nous rappellant ce qui constitue le Material Design, ces éléments (paper, touch first, couleur textuelle, animation, bouton flottant, ...). Il nous a ensuite fait une démonstration de Polymer (qui vient de sortir en version 1.0), quelques petits bower install plustard et c'est parti, on charge polymer dans la balise script et on peut utiliser les paper-card en utilisant des ``<link rel="import" href`` pour les importer et ensuite directement dans notre code html. La bibliothéque de base de Polymer est déjà bien rempli mais il est également possible de créer ses propres cards et de bien s'amuser avec le Material Design. Polymer donne vraiment envie, pouvoir créer une interface Material Design en quelque coups d'import, ça promet de belles choses.
+ 
+## Le [Reactive Programming](https://speakerdeck.com/hugocrd/dealing-with-streams-using-rxjs) par Hugo Cordier
+Les événements en Javascript sont l'un des composants important, d'autant plus maintenant avec les frameworks js. Hugo Cordier nous a fait un petit récap' de ce que sont les événements et les promesses, puis présenté RxJs qui permet de faire du traitement de groupe d'événements (événements multiples, au cours du temps, faire des map et des filter sur les événements), en bref un vrai petit couteau suisse de la gestion des événements en Javascript (il s'agit d'une librairie donc vous pouvez vous amusez avec à peut près tout: node, angular, backbone). Il nous a ensuite fait une démonstration de cas concret de RxJs comme par exemple de la recherche en temps réelle (n'hésitez pas à jeter un oeil à ses codepens pour les démos, ils sont présents dans la présentation).
```

Comment:
When not all uppercase, CodePen is written in [PascalCase](https://en.wikipedia.org/wiki/CamelCase).